manifests/n/Neovim/Neovim/0.9.4/Neovim.Neovim.locale.en-US.yaml

```diff
@@ -0,0 +1,28 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Neovim.Neovim
+PackageVersion: 0.9.4
+PackageLocale: en-US
+Publisher: neovim.io
+PublisherUrl: https://neovim.io
+PublisherSupportUrl: https://github.com/neovim/neovim/issues
+PackageName: Neovim
+PackageUrl: https://neovim.io
+License: Apache-2.0 or Vim
+LicenseUrl: https://github.com/neovim/neovim/blob/master/LICENSE.txt
+ShortDescription: Vim-fork focused on extensibility and usability
+Moniker: nvim
+Tags:
+- editor
+- nvim-qt
+- text
+- vi
+- vim
+ReleaseNotes: |-

```

Comment:
This is a maintenance release, focused on bug fixes and performance improvements. However, There are included features related to TUI and `:terminal.`